_pytest/config.py

```diff
@@ -364,6 +364,9 @@ def _importconftest(self, conftestpath):
                 _ensure_removed_sysmodule(conftestpath.purebasename)
             try:
                 mod = conftestpath.pyimport()
+                if hasattr(mod, 'pytest_plugins') and len(self._conftest_plugins) > 0:
```

Comment:
@RonnyPfannschmidt is using `len(self._conftest_plugins) > 0` to identify the root conftest the correct approach, regardless where the collection started? Just making sure.

Reply:
its not, i'll take a while to identify the right approach for the initial contests

Reply:
Can you give an example of a situation in which the top level conftest won't be the first collected? I wasn't able to find one.

Reply:
@brianmaissy the situation i wondering about is where there is no toplevel conftest to begin with

Reply:
Ah I see. Any conftest whose directory is not an ancestor of all collected tests. So we can determine this only after collection finished.

Reply:
Or we could say that only a conftest located in the rootdir can use `pytest_plugins`. But that might be too restrictive.

Reply:
So for sure my implementation yields false negatives. You could also say it yields false positives if you consider this directory structure valid:

a/
  conftest.py (defining pytest_plugins)
  b/
    conftest.py (defining pytest_plugins)
    test_it.py

Reply:
in order to sort it out i believe a check if pytest is configured fits

early conftests are imported and used while the `pytest_configure` step did not yet happen,
later conftests are imported as part of collection after `pytest_configure` did happen